lib/bots/versionbot.ts

```diff
@@ -84,12 +84,24 @@ interface FSError {
     message: string;
 }
 
+interface VersionBotConfiguration extends GithubBotConfiguration {
+    githubbot: {
+        versionbot: {
+            minimum_approvals?: number;
+            approved_reviewers?: string[];
+            maintainers?: string[];
+        };
+    };
+}
+
 // The VersionBot is built on top of GithubBot, which does all the heavy lifting and scheduling.
 // It is designed to check for valid `versionist` commit semantics and alter (or merge) a PR
 // accordingly.
 export class VersionBot extends GithubBot.GithubBot {
     // Flowdock Adapter
     private flowdock: FlowdockAdapter;
+    private mergeLabel = 'procbots/versionbot/ready-to-merge';
+    private ignoreLabel = 'procbots/versionbot/no-checks';
```

Comment:
JavaScript I got into, fine. TypeScript is still occasionally causing me headaches. I'm still trying to break out of the C++ mentality, so up the top it'd have turned into a static (or worse a global), hence keeping it in the class.
I'll get there eventually, I promise!